### PR TITLE
Melkehitys-1023

### PR DESCRIPTION
**Pull request by natlibfi-jonollil**
### Update:
Package, test material, callback to Emitter based and stream handling to more atomic

### Local test settings:
- melinda-record-import-ap #melkehitys-1020 build locally 
- quay.io/natlibfi/melinda-record-import-controller:test
- melinda-record-import-importer-dummy #melkehitys-1023 build locally
- melinda-record-import-transformer-dummy #melkehitys-1023 build locally

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -26,15 +26,58 @@
...
 
 const {runCLI} = Transformer;
 
 run();
 
 async function run() {
-	const validateCallback = createValidator();
-	runCLI({name: 'melinda-record-import-transformer-dummy', transformCallback, validateCallback});
+	const transformerSettings = {
+		name: 'melinda-record-import-transformer-helmet',
```
Comment (ghost) [inline src/cli.js]: Fix name

**Review by ghost — CHANGES_REQUESTED** [review]
Replace Helmet-specific texts (README has this too).

**Review by ghost — CHANGES_REQUESTED** [review]
Remove this from README: 
```
 Consumes records as represented by Helmet's [Sierra ILS](https://sandbox.iii.com/iii/sierra-api/swagger/index.html)
```

**Comment by natlibfi-jonollil** [comment]
If stream is given to dummy transformer, all records will be validated as valid and sent to importter.
Also all streamed records are replaced whit dummy records, only amount of streamed records is valid.

**Review by ghost — CHANGES_REQUESTED** [review]
This needs the same changes as Helmet transformer (These should be implemented after Helmet transformer works correctly).

**Review by ghost — CHANGES_REQUESTED** [review]
Should be fixed when implementation is ready in API, commons and Helmet-transformer.

**Review by ghost — CHANGES_REQUESTED** [review]
[@natlibfi/melinda-record-import-commons](https://www.npmjs.com/package/@natlibfi/melinda-record-import-commons/v/7.0.0) has been published. Update dependency to point to that version in NPM.

**Review by ghost — APPROVED** [review]

File: .drone.yml [hunk 2]
```diff
@@ -3,8 +3,13 @@ name: default
 steps:
 - name: generate-tags
   image: quay.io/natlibfi/drone-gen-tags
-- name: build
+- name: test-init
+  image: busybox
+  commands:
+  - chmod o+w /drone/src
+- name: test
   image: node:8
```
Comment (ghost) [inline .drone.yml]: Change to `node:10`.

**Review by ghost — CHANGES_REQUESTED** [review]
Files `test.json` and `test20.json` were probably included accidentally in version control, so please remove them.

**Review by ghost — APPROVED** [review]

**Pull request merged**